### PR TITLE
Product page with filter functionality

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made:
- Added products
- Added filters and sort options
- Ignore cart and wishlist page code. These code added so that netlify build success
Screenshot (179)

### CODE REVIEW

**Review by romabulani — APPROVED** [review]
- Liked that you have broken the UI into very small components like radio buttons and checkboxes
- Good Code clarity

Just a small suggestion, while writing the name for the folders, it is recommended to use all small case letters like pages, components and so on. Its not a big deal, if you have time then only refactor it,

**Comment by SanketDhabarde** [comment]
> * Liked that you have broken the UI into very small components like radio buttons and checkboxes
> * Good Code clarity
> 
> Just a small suggestion, while writing the name for the folders, it is recommended to use all small case letters like pages, components, and so on. It's not a big deal, if you have time then only refactor it,

Ok, I will make changes in the next pr. thank you for the suggestion.☺

File: src/Pages/Cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import CartCard from "../../Components/CartCard/CartCard";
+import Nav from "../../Components/Nav/Nav";
```
Comment (rohanmathur91) [inline src/Pages/Cart/Cart.jsx:3]: Import from index.js of your components folder

File: src/Pages/Wishlist/Wishlist.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Nav from '../../Components/Nav/Nav';
+import ProductCard from '../../Components/ProductCard/ProductCard';
```
Comment (rohanmathur91) [inline src/Pages/Wishlist/Wishlist.jsx:3]: Same as above

**Review by rohanmathur91 — APPROVED** [review]
Added few comments.

- Liked that you have a reusable component for the checkbox and radio
- Having a custom hook for data fetching is good idea
- Liked that each util function is separated  and imported were needed.

File: src/Pages/Cart/Cart.jsx (hunk 1 above)
Comment (SanketDhabarde) [reply src/Pages/Cart/Cart.jsx:3]: Thanks, Rohan. I will do it. when implementing the cart and wishlist

**Comment by SanketDhabarde** [comment]
> Added few comments.
> 
> * Liked that you have a reusable component for the checkbox and radio
> * Having a custom hook for data fetching is good idea
> * Liked that each util function is separated  and imported were needed.

Thank you Rohan. ☺

**Pull request merged**